c_binding/lsm_convert.cpp

```diff
@@ -156,7 +156,7 @@ lsm_disk *value_to_disk(Value & disk)
 
         if ((rc != NULL) && std_map_has_key(d, "disk_location") &&
             (d["disk_location"].asC_str()[0] != '\0')) {
-            
+
```

Comment:
Any particular reason for the newline? It's not present in the new attribute checks you just added...